README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/expectedbehavior/acts_as_archival.svg?branch=master)](https://travis-ci.org/expectedbehavior/acts_as_archival)
```

Comment:
Do you want me to do it like Rails maybe? https://github.com/rails/rails#code-status or like paper_trail (see below).

As for the number of badges, I guess it's up to you as the maintainer, since I can't add more integrations on my own.
[rpush has a lot](https://github.com/rpush/rpush), [paper_trail](https://github.com/airblade/paper_trail#papertrail) has two.